src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -272,96 +272,102 @@ const formConfig = {
           path: 'benefits-information',
           title: 'Benefits Information',
           uiSchema: {
-            programs: {
```

Comment:
The schema structure was causing a 422. Nesting programs, assistance, and school within educationDetails resolves the error.